### PR TITLE
Cores configuration

### LINKED ISSUE #475: GOMAXPROCS customization
https://github.com/openshift/origin/pull/3034 makes OpenShift default to setting GOMAXPROCS to the number of available CPUs. Admins can tune this by setting the GOMAXPROCS envvar.

### PR DESCRIPTION
**Pull request by 0xmichalis**
Fixes #475

@liggitt PTAL

### CODE REVIEW

File: getting_started/administrators.adoc [hunk 1]
```diff
@@ -33,7 +33,16 @@ up, you can continue by choosing one of the following installation methods.
 
 == Installation Methods
 
-Choose one of the following installation methods that works best for you.
+Before moving forward, you can choose the number of cores the OpenShift server should use:
++
+----
+$ export GOMAXPROCS=n
```
Comment (liggitt) [inline getting_started/administrators.adoc]: They can choose this post-install. It also needs to be set in whatever environment the server will run in (so if they run in a docker container, it has to be set in the docker container, etc).

**Comment by kargakis** [comment]
@liggitt updated, not sure about the asciidoc markdown though...

**Comment by adellape** [comment]
@kargakis @liggitt Those sections of "Getting Started for Admins" currently are conditionalized to only show up for the Origin doc build. Assuming that this also applies to OSE 3.0 GA, could we instead move this info into some place more generic, like:

https://docs.openshift.org/latest/admin_guide/install/prerequisites.html#system-requirements

The `export` instruction could appear for OSE+Origin, and the `-e` instruction could be conditionalized to appear only as an Origin-applicable alternative (in case they'll be running it as a container).

And: applicable for both master and node hosts, I take it?

**Comment by liggitt** [comment]
yeah, should be more widely surfaced. applies to both master and node

**Comment by kargakis** [comment]
How is cores configuration part of Prerequisites/System requirements? As @liggitt said, people can configure this post-install and usually that's the place they will.

**Comment by adellape** [comment]
@kargakis Back from the grave. See https://github.com/openshift/openshift-docs/pull/860 which builds on this.

> How is cores configuration part of Prerequisites/System requirements? As @liggitt said, people can configure this post-install and usually that's the place they will.

I think it makes sense there as a spot where folks would think to eyeball resource/spec requirements. I don't know where else someone would think to check on that if not there (unless we get a Tuning Guide or something equivalent at some point), so makes sense to me to catch em there w/ this info. Also, the OSE build[1] of the Admin Getting Started doesn't have all the stuff in it that the Origin build[2] does, so it would get lost there for OSE.

[1] https://docs.openshift.com/enterprise/3.0/getting_started/administrators.html
[2] https://docs.openshift.org/latest/getting_started/administrators.html

**Comment by adellape** [comment]
And yes, both "Getting Started" topics for OSE are ridiculous right now. :person_frowning:

**Comment by kargakis** [comment]
> a spot where folks would think to eyeball resource/spec requirements. I don't know where else someone would think to check on that if not there

@adellape makes sense

**Comment by adellape** [comment]
Closing per https://github.com/openshift/openshift-docs/pull/860.

**Pull request closed without merging**